tests/conftest.py

```diff
@@ -32,25 +33,52 @@ def remove_files_and_folders(path, file_structure):
                 print(f"PermissionError: {root}")
 
 
-@pytest.fixture
-def test_db():
-    """This fixture is used for testing in general to setup the test database,
-    perform the test, and clean the database after each test.
-    It is used by other fixtures to set up and clean the test_database"""
+@pytest.fixture(
+    autouse=True, scope="session"
+)  # This fixture is only run once per session
+def updatedSVN():
+    parse_config(
+        "config.toml"
+    )  # Set the database root, system folder, based on the config file
+    set_site_name("charleston_test")  # set the site name to the test database
```

Comment:
Maybe we should change this name as well. Either it should be a general testing database, or maybe we want to make this configurable so that we can have multiple test cases and thus also multiple test databases? Not sure what the right answer is but maybe something to think about

Reply:
We should definitely have a discussion about what to do with this test database. @CFBaptista mentioned a mock database that is saved as data in the testfiles, instead of the actual database folder. (like what you did in your parameterized tests) I think that might be a good long term goal, but for now this is okay I think.

Currently, if we want to add additional tests databases, we can just make another test_db fixture and use a different site_name.